app/src/main/java/mobi/acpm/inspeckage/util/PackageDetail.java

```diff
@@ -326,7 +326,7 @@ public String getDataDir() {
 
     public String getGIDs() {
         String gidList = "";
-        if (mPInfo.gids != null) {
+		if (mPInfo.gids != null && mPInfo.gids.length != 0) {
```

Comment:
This line looks like it used a tab character for indentation, which is inconsistent with the surrounding source code (you can see in the diff how the indents are uneven and make it harder to read). I'm not a core contributor but would recommend switching it back to spaces for better readability.